src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,50 @@
+package duke.task;
+
+import java.time.LocalDateTime;
+
+/**
+ * A task that start at a specific date/time and ends at a specific date/time.
+ */
+public class Event extends Task {
+    private final LocalDateTime datetime;
```

Comment:
should this be called dateTime instead?
_" Variable names must be in camelCase."_